sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/DataSourceSuite.scala

```diff
@@ -214,4 +216,6 @@ class MockFileSystem extends RawLocalFileSystem {
   override def globStatus(pathPattern: Path): Array[FileStatus] = {
     mockGlobResults.getOrElse(pathPattern, Array())
   }
+
+  override def getUri: URI = URI.create("mockFs://mockFs/")
```

Comment:
is this change needed?

Reply:
Yes, if we don't override this method, then path check inside `fs.makeQualified(path)` will fail, because it expects path with `file://` scheme (`MockFileSystem` inherits `RawLocalFileSystem`)